config.c

```diff
@@ -175,6 +176,20 @@ static int apply_config_option(struct mako_config *config,
 		free(config->output);
 		config->output = strdup(value);
 		return 0;
+	} else if (strcmp(name, "urgency") == 0) {
```

Comment:
There's probably a better way to do this. Let me know if you have any ideas.

Reply:
Hmm, is it possible to have a `sort` option instead, as described in https://github.com/emersion/mako/issues/24#issuecomment-389888975 ? Let me know if you have better ideas :)

Reply:
I added these options. You can specify `--sort` mutiple times with `+/-time` and `+/-priority`

Reply:
Hmm? Correct me if I'm mistaken, but this code doesn't seem to do this. It seems to do things like `--time asc --urgency des` instead.

Reply:
Sorry! Forgot to push. :)